tools/block-generator/Makefile

```diff
@@ -22,15 +23,29 @@ cleanup: clean-reports
 enter-pg:
 	docker exec -it generator-test-container psql -U algorand -d generator_db
 
-run-runner:
+clean:
+	rm -f block-generator
+
+block-generator: clean
+	go build
+
+run-runner: block-generator
 	./block-generator runner --conduit-binary ./conduit \
-	--log-level trace \
 	--keep-data-dir \
 	--test-duration $(DURATION) \
-	--log-level trace \
+	--conduit-log-level trace \
 	--postgres-connection-string "host=localhost user=algorand password=algorand dbname=generator_db port=15432 sslmode=disable" \
 	--scenario $(SCENARIO) \
 	$(RESETDB) \
+	$(VERBOSE) \
 	--report-directory $(REPORTS)
 
+pre-git-push:
+	mv _go.mod go.mod
```

Comment:
what are these _go.mod files?

Reply:
My VS Code was having a hard time following references to type definitions outside of the current file because of the go module we introduced to block-generator. So I find it easier to develop with my `go.mod` and `go.sum` renamed `_go.mod` and `_go.sum`. I want to make sure to rename them and `make tidy` before I commit so I have these hack `pre-git-push` and `post-git-push` make targets. Will be glad to remove if that's too annoying.